themes/default/content/what-is/what-is-platform-engineering.md

```diff
@@ -2,42 +2,49 @@
 
-The responsibilities of a platform engineer may include the following:
+## What are some of the principles behind platform engineering?
 
-* Providing development teams with self-service infrastructure tooling and golden paths to production
+Regardless of implementation details or specific methods, there are some simple principles that many platform engineering teams follow. All of these principles are driving toward maximizing the use of the cloud at scale across an organization while being secure and compliant and enabling developers and develpoment teams to ship faster.
 
-* Defining and creating standardized cloud architectures used by development teams, typically using an [infrastructure as code framework](/what-is/what-is-infrastructure-as-code/)
+* **Simple and powerful user experience**: Build curated experiences that empower developers by meeting them at their level of expertise. Use a variety of approaches to provide an ideal user/developer experience, including infrastructure code libraries (reusable pieces of code), infrastructure CLIs, internal developer portals (IDPs), or shared IaC templates.
+* **Automation as default**: Automation reduces devastating errors. Don't leave anything to ClickOps. Every change to infrastructure must be run through tests before rolling into production. No change should be untraceable. All infrastructure from resources, configurations, environments, and secrets are tracked in version control. Everything from infrastructure provisioning to control plane orchestrations needs to be programmable.
+* **Full visibility on everything**: Log, monitor, and observe all infrastructure for greater operational control. Optimize against unnecessary costs.
+* **Security as a foundation**: Security and compliance guardrails need to exist for everything. Fine-grained access controls should exist for every piece of infrastructure. Prefer dynamic, short-lived credentials over long-lived, static credentials with seamless integration into development workflows.
+* **Well architected by design**: Decouple the application complexity from the infrastructure complexity, reducing the blast radius of incidents while increasing resiliency. Construct shareable infrastructure components that are built for high availability and low operational maintenance. Operate seamlessly across heterogeneous environments while implementing best-in-class infrastructure.
 
-* [Automating infrastructure and operational tasks](/automation/) using tools such as infrastructure as code and continuous integration/delivery (CI/CD) pipelines
+## What are the key components of platform engineering?
```

Comment:
what is really throwing me off here is we talk about principles and then components. both which are parts that make up "something". i get the two words mean something different, but it just feels somehow wrong. 

i guess where i am struggling is i can't seem to make the three things in the components section fit nicely into the messaigng framework document

Reply:
I've removed this section in the latest commit.